gymnasium/envs/mujoco/humanoid_v5.py

```diff
@@ -0,0 +1,516 @@
+    | 8  | 2      | abdomen_x |
+    | 9  | 3      | right_hip_x |
+    | 10 | 4      | right_hip_z |
+    | 11 | 5      | right_hip_y |
+    | 12 | 6      | right_knee |
+    | 13 | 7      | left_hip_x |
+    | 14 | 8      | left_hiz_z |
+    | 15 | 9      | left_hip_y |
+    | 16 | 10     | left_knee |
+    | 17 | 11     | right_shoulder1 |
+    | 18 | 12     | right_shoulder2 |
+    | 19 | 13     | right_elbow|
+    | 20 | 14     | left_shoulder1 |
+    | 21 | 15     | left_shoulder2 |
+    | 22 | 16     | left_elfbow |
+
+    The (x,y,z) coordinates are translational DOFs while the orientations are rotational
+    DOFs expressed as quaternions. One can read more about free joints on the
+    [Mujoco Documentation](https://mujoco.readthedocs.io/en/latest/XMLreference.html).
+
+    **Note:** Humanoid-v4 environment no longer has the following contact forces issue.
+    If using previous Humanoid versions from v4, there have been reported issues that using a Mujoco-Py version > 2.0
+    results in the contact forces always being 0. As such we recommend to use a Mujoco-Py
+    version < 2.0 when using the Humanoid environment if you would like to report results
+    with contact forces (if contact forces are not used in your experiments, you can use
+    version > 2.0).
+
+
+    ## Rewards
+    The reward consists of three parts:
+    - *healthy_reward*:
+    Every timestep that the Humanoid is alive (see section Episode Termination for definition),
+    it gets a reward of fixed value `healthy_reward`.
+    - *forward_reward*:
+    A reward for moving forward,
+    this reward would be positive if the Humanoid moves forward (in the positive $x$ direction / in the right direction).
+    $w_{forward} \times \frac{dx}{dt}$, where
+    $dx$ is the displacement of the center of mass ($x_{after-action} - x_{before-action}$),
+    $dt$ is the time between actions, which depends on the `frame_skip` parameter (default is 5),
+    and `frametime` which is 0.001 - so the default is $dt = 5 \times 0.003 = 0.015$,
+    $w_{forward}$ is the `forward_reward_weight` (default is $1.25$).
+    The center of mass calculation is defined in the `.py` file for the Humanoid.
+    - *ctrl_cost*:
+    A negative reward to penalize the Humanoid for taking actions that are too large.
+    $w_{control} \times \\|action\\|_2^2$,
+    where $w_{control}$ is `ctrl_cost_weight` (default is $0.1$).
+    If there are *nu* actuators/controls, then the control has shape `nu x 1`.
+    - *contact_cost*:
+    A negative reward to penalize the Humanoid if the external contact forces are too large.
+    $w_{contact} \times clamp(contact\\_cost\\_range, \\|F_{contact}\\|_2^2)$, where
+    $w_{contact}$ is `contact_cost_weight` (default is $5\times10^{-7}$),
+    $F_{contact}$ are the external contact forces (see `cfrc_ext` section on observation).
+
+    The total reward returned is ***reward*** *=* *healthy_reward + forward_reward - ctrl_cost - contact_cost*
```

Comment:
I would move this to the top of the section with the individual reward terms defined below

Reply:
Which do you prefer
# option 1

The total reward is: ***reward*** *=* *healthy_reward + forward_reward - ctrl_cost - contact_cost*.
...

# option 2
The total reward is: $r = r_{healthy} + r_{fordward} - c_{control} - c_{contact}$.
...

Reply:
option 1